tahoma_api/client.py

```diff
@@ -107,7 +107,7 @@ async def register_event_listener(self) -> str:
         """
         response = await self.__post("events/register")
         listener_id = response.get("id")
-        self.event_listener = listener_id
+        self.event_listener_id = listener_id
```

Comment:
IMO the variable listener_id is useless. But it's ok.

Reply:
@tetienne I agree. However linter was complaining about the fact that it could be str or None. Instead of str.

Is there a way to annotate that response.get("id") will return a string always

Reply:
`.get` will return None if the key can not be found. Try with response["id"].

Reply:
@tetienne same. I will keep it like this for now.

```
tahoma_api/client.py:111: error: Incompatible return value type (got "Optional[str]", expected "str")
```